docker-storage-setup.sh

```diff
@@ -888,6 +895,17 @@ else
   fi
 fi
 
+# Main Script
+
+if [ $# -gt 0 ]; then
+    if [ "$1" == "reset" ]; then
```

Comment:
In fact, initially don't do anything. We will just document it. Also we can document in man page that preferred way to do this is by calling "atomic storage reset".

Also if docker metadata is not cleaned and next time one starts docker, docker-storage-setup will fail saying there is already devicemapper metadata in dokcer and it will refuse to setup new thin pool.

So I really don't see a need of an option like `internal-only-for-atomic-reset`